app/src/main/java/com/example/tasky/feature_agenda/data/mapper/AgendaMapper.kt

```diff
@@ -0,0 +1,73 @@
+package com.example.tasky.feature_agenda.data.mapper
+
+import android.os.Build
+import androidx.annotation.RequiresApi
+import com.example.tasky.feature_agenda.data.remote.dto.AttendeesDto
+import com.example.tasky.feature_agenda.data.remote.dto.EventDto
+import com.example.tasky.feature_agenda.data.remote.dto.PhotoDto
+import com.example.tasky.feature_agenda.data.remote.dto.ReminderDto
+import com.example.tasky.feature_agenda.data.remote.dto.TaskDto
+import com.example.tasky.feature_agenda.domain.model.Attendee
+import com.example.tasky.feature_agenda.domain.model.Event
+import com.example.tasky.feature_agenda.domain.model.Photo
+import com.example.tasky.feature_agenda.domain.model.Reminder
+import com.example.tasky.feature_agenda.domain.model.Task
+import java.time.Instant
+import java.time.LocalDateTime
+import java.time.ZoneId
+
+@RequiresApi(Build.VERSION_CODES.O)
+fun Long.toTime(): LocalDateTime {
+    val instant = Instant.ofEpochMilli(this)
+    return LocalDateTime.ofInstant(instant, ZoneId.systemDefault())
+}
```

Comment:
This function looks like it contains a bug. `Instant.ofEpochMilli` expects the provided millis in UTC time. However, when converting them to `LocalDateTime`, you interpret them as the local user's time zone. If the user is in a different time zone than UTC, the time will be incorrect by a few hours.

To fix this, you need to provide the UTC ZoneId instead: `ZoneId.of("UTC")`
Also, to make this clearer for the callers of the function, I'd clarify it in the name and call this function `asUtcTimestampToLocalDateTime()` or similar.

PS: If you want to display the time in the user's time zone, you'd also need to convert it after calling this function.